src/plugins/data_source_management/public/service/data_source_management_service.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
```

Comment:
don't think we need this file, please try remove it

Reply:
Done